src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,25 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <>
+    <ul className="movies">
+      {movies.map(movie => (
+        <li key={movie.imdbId}>
+          <MovieCard
+            title={movie.title}
+            description={movie.description}
+            imgUrl={movie.imgUrl}
+            imdbUrl={movie.imdbUrl}
+          />
```

Comment:
```suggestion
          <MovieCard movie={movie} />
```
You can pass the whole `movie` object to the card